packages/@react-stately/select/src/useSelectState.ts

```diff
@@ -60,7 +59,7 @@ export function useSelectState<T>(props: SelectProps<T>): SelectState<T>  {
     setFocusStrategy,
     toggle(focusStrategy = 'first') {
       setFocusStrategy(focusStrategy);
-      setOpen(isOpen => !isOpen);
+      setOpen(!isOpen);
```

Comment:
Typescript started complaining, also not entirely sure where toggle gets used still digging